src/phoenix/server/templates/index.html

```diff
@@ -23,6 +23,9 @@
             // injected into the client before React runs
             value: Object.freeze({
                 hasCorpus: Boolean("{{has_corpus}}"),
+                minDist: parseFloat("{{min_dist}}"),
+                nNeighbors: parseInt("{{n_neighbors}}"),
+                nSamples: parseInt("{{n_samples}}"),
```

Comment:
```suggestion
                umap: {
                    minDist: parseFloat("{{min_dist}}"),
                    nNeighbors: parseInt("{{n_neighbors}}"),
                    nSamples: parseInt("{{n_samples}}"),
                }
```

nesting it under umap might help with making it a bit more obvious what these params are associated with